plugins/it.xsemantics.dsl/src/it/xsemantics/dsl/XsemanticsModelGenerator.xtend

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
```

Comment:
Hi,
Sorry I still don't understand... I've just tried the current version of Xsemantics, with something like this:

```
system my.test.System1

copyright 
"Foo
and bar"
```

and it correctly generates this Java file:

```
/**
 * Foo
 * and bar
 */
package my.test;
```

which looks good to me, without trailing spaces...
Moreover, in the Xsemantics file, I don't have to specify comments explicitly.

@JensN4 could you please comment on that?

I'd prefer not to have to specify comment syntax explicitly in the copyright field, since this also breaks existing Xsemantics files.

From what I understand, with the implementation of this PR you can decide the shape of your copyright headers right? While now you rely on the one provided by Xbase implementation, is that right?

If you really want to have full control on the header, then I'd suggest to add another syntax something like "fileHeader", while leaving the "copyright" specification as it is now... but I would do that in another PR...